src/pages/projects/index.page.tsx

```diff
@@ -114,31 +118,23 @@ function Projects() {
   const { data: searchPages, isLoading: isLoadingSearch } =
     useProjectsSearch(searchText)
 
-  // When we scroll within 200px of our loadMoreContainerRef, fetch the next page.
-  useEffect(() => {
-    if (loadMoreContainerRef.current && selectedTab !== 'trending') {
-      const observer = new IntersectionObserver(
-        entries => {
-          if (entries.find(e => e.isIntersecting) && hasNextPage) {
-            fetchNextPage()
-          }
-        },
-        {
-          rootMargin: '200px',
-        },
-      )
-      observer.observe(loadMoreContainerRef.current)
-
-      return () => observer.disconnect()
-    }
-  }, [selectedTab, fetchNextPage, hasNextPage])
+  const [visible] = useLoadMoreContent({
```

Comment:
Just one **nit-pick**- do you think `visible` might be better named `scrolledTo` or `scrolledToBottom` or something along those lines?

Reply:
agreed million percent. thanks for this nit-pick!